redis.conf

```diff
@@ -1576,6 +1576,17 @@ lua-time-limit 5000
 #
 # cluster-allow-reads-when-down no
 
+# Cluster link send buffer limit is the limit on the memory usage of an individual
+# cluster bus link's send buffer in bytes. Cluster links would be freed if they exceed
+# this limit. This is to primarily prevent send buffers from growing unbounded on links
+# toward slow peers (E.g. PubSub messages being piled up).
+# This limit is disabled by default. Enable this limit when 'mem_cluster_links' INFO field
+# and/or 'sendbufalloc' entries in the 'CLUSTER LINKS` command output continuously increase.
+# Minimum limit of 1gb is recommended so that cluster link buffer can fit in at least a single
+# PubSub message by default. (client-query-buffer-limit default value is 1gb)
```

Comment:
the documented default, and the default value in the conf file are outdated (the one in config.c is ok)

Reply:
While I'm here I guess I'll fix this.